dist/knockout.validation.d.ts

```diff
@@ -0,0 +1,446 @@
+        };
+        validationOptions: {
+            init(element: HTMLElement, valueAccessor: () => ko.MaybeSubscribable<validation.ValidationConfiguration>, allBindingsAccessor: ko.AllBindings, viewModel: any, bindingContext: ko.BindingContext<any>): void;
+        };
+    }
+
+    export interface Extenders {
+        /**
+         * This is for creating custom validation logic on the fly.
+         * 
+         * @example
+         *      var test = ko.observable("something").extend({
+         *          validation: {
+         *              validator: function (val, someOtherVal) {
+         *                  return true;
+         *              },
+         *              message: "Something must be really wrong!",
+         *              params: true
+         *          }
+         *      });
+         */
+        validation<T extends ko.Subscribable<any>>(target: T, options: validation.ValidationRule | validation.ValidationRule[]): T & validation.ObservableValidationExtension;
+        /**
+         * This is the extender that makes a Knockout Observable also 'Validatable'
+         * 
+         * @example
+         *  // This will ensure that the Observable object is setup properly to respond to rules
+         *  var test = ko.observable("something").extend({ validatable: true });
+         * 
+         *  // This will remove the validation properties from the Observable object should you need to do that.
+         *  test.extend({ validatable: false });
+         */
+        validatable<T extends ko.Subscribable<any>>(target: T, options: boolean | validation.ExtenderValidatableOptions): T & validation.ObservableValidationExtension;
+
+        required<T extends ko.Subscribable<any>>(target: T, options: boolean | validation.ValidationRuleExtenderParams<boolean>): T & validation.ObservableValidationExtension;
+        min<T extends ko.Subscribable<any>>(target: T, options: number | validation.ValidationRuleExtenderParams<number>): T & validation.ObservableValidationExtension;
+        max<T extends ko.Subscribable<any>>(target: T, options: number | validation.ValidationRuleExtenderParams<number>): T & validation.ObservableValidationExtension;
+        minLength<T extends ko.Subscribable<any>>(target: T, options: number | validation.ValidationRuleExtenderParams<number>): T & validation.ObservableValidationExtension;
+        maxLength<T extends ko.Subscribable<any>>(target: T, options: number | validation.ValidationRuleExtenderParams<number>): T & validation.ObservableValidationExtension;
+        pattern<T extends ko.Subscribable<any>>(target: T, options: RegExp | validation.ValidationRuleExtenderParams<RegExp>): T & validation.ObservableValidationExtension;
+        step<T extends ko.Subscribable<any>>(target: T, options: RegExp | validation.ValidationRuleExtenderParams<RegExp>): T & validation.ObservableValidationExtension;
```

Comment:
This line should match min, not pattern. It should be:
        step<T extends ko.Subscribable<any>>(target: T, options: number | validation.ValidationRuleExtenderParams<number>): T & validation.ObservableValidationExtension;

Reply:
Done!